docs/index.asciidoc

```diff
@@ -173,7 +173,7 @@ This is useful in case the time zone cannot be extracted from the value,
 and is not the platform default.
 If this is not specified the platform default will be used.
 Canonical ID is good as it takes care of daylight saving time for you
-For example, `America/Los_Angeles` or `Europe/France` are valid IDs.
+For example, `America/Los_Angeles` or `Europe/Paris` are valid IDs.
```

Comment:
:+1: I can confirm that `Europe/Paris` is in recent editions of the timezonedb https://en.wikipedia.org/wiki/List_of_tz_database_time_zones